i18n/extra/es.json

```diff
@@ -84,7 +84,10 @@
         "SMW_NS_CONCEPT": "Concepto",
         "SMW_NS_CONCEPT_TALK": "Concepto_discusión"
     },
-    "namespaceAliases": [],
+    "namespaceAliases": {
+      "Atributo": "SMW_NS_PROPERTY",
```

Comment:
@ivanhercaz A nitpick. The indentation is not the same as for the other elements in this json file.

Reply:
Ups, I will fix it and I will notice you, @kghbln :)

Reply:
I think that I fixed it, @kghbln.